src/connectors/numeric-menu/__tests__/connectNumericMenu-test.ts

```diff
@@ -1131,6 +1131,68 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/numeric-men
   });
 
   describe('getWidgetRenderState', () => {
+    it('returns the widget render state before init', () => {
+      const rendering = jest.fn();
+      const makeWidget = connectNumericMenu(rendering);
+      const widget = makeWidget({
+        attribute: 'numerics',
+        items: [
+          { label: 'below 10', end: 10 },
+          { label: '10 - 20', start: 10, end: 20 },
+          { label: 'more than 20', start: 20 },
+        ],
+      });
+
+      const helper = jsHelper(createSearchClient(), '');
+      helper.search = () => helper;
```

Comment:
I think it's not necessary to initialize `helper.search()` since calling `createSearchClient()` will do it, I might be wrong tho

Reply:
you're right! https://github.com/algolia/instantsearch.js/pull/4579/commits/47e83d61c41975a1d57a17bfc978cda92b9cf9d5